### ISSUE TITLE
Screen reader does not read tooltip when <d2l-button> is target

### ISSUE THREAD

**Issue by CodeBaboon** [BrightspaceUI/tooltip#17]
Due to issues around focus within web components the actual focused element of a `<d2l-button>` is not the same element that is associated to the `<d2l-tooltip>` within the DOM. Therefor when focused the screen reader does not read out the contents of the tooltip when it is assigned as the control for `aria-describedby` on the `<d2l-button>`.

Possible workaround is to turn the `tooltip` into a Polymer 2 mixin.

**Comment by jstefaniuk-d2l** [comment]
You may be able to fix this by adding an `aria-live` property on the tooltip (although then it'd read on mouseover as well)
